settings.py

```diff
@@ -2256,6 +2256,16 @@ def _determine_couch_databases(couch_databases):
     'test-pna': 'custom.yeksi_naa_reports',
 }
 
+OTHER_DOMAINS_WITH_CUSTOM_MODULES = [
+    'fri',
+    'hope',
+    'logistics',
+    'icds',
+    'hki',
+    'nikshay',
```

Comment:
if this is meant to match enikshay we can remove it as well

Reply:
I'm not too sure what it is, but it should match this domain https://www.commcarehq.org/a/nikshay/dashboard/project/